HedgeModManager/Resources/Codesv2/SonicFrontiers.hmm

```diff
@@ -7874,7 +7874,7 @@ Patch "Disable Tutorial Messages" in "Gameplay" by "Hyper" does "Disables all in
     );
 }
 
-Patch "Disable Umbrella Camera Lock-On" in "Camera" by "Hyper" does "Disables the lock-on camera when the Umbrella enemy in Ares jumps into the sky."
+Patch "Disable Umbrella Camera Lock-On" in "Camera" by "Hyper" does "Disables the lock-on camera when the Umbrella enemy jumps into the sky."
```

Comment:
What's the point in removing this? It makes it more clear which enemy is being referred to, seeing as you have to dig through concept art for the enemy names.